extensions/wikia/ArticleVideo/bower_components/html5-skin/js/views/adScreen.js

```diff
@@ -0,0 +1,194 @@
+/********************************************************************
+  AD SCREEN
+*********************************************************************/
+var React = require('react'),
+    CONSTANTS = require('../constants/constants'),
+    AdPanel = require('../components/adPanel'),
+    ControlBar = require('../components/controlBar'),
+    ClassNames = require('classnames'),
+    Utils = require('../components/utils'),
+    ResizeMixin = require('../mixins/resizeMixin'),
+    Icon = require('../components/icon');
+
+var AdScreen = React.createClass({
+  mixins: [ResizeMixin],
+
+  getInitialState: function() {
+    this.isMobile = this.props.controller.state.isMobile;
+    return {
+      controlBarVisible: true,
+      timer: null
+    };
+  },
+
+  componentDidMount: function () {
+    //for mobile or desktop fullscreen, hide control bar after 3 seconds
+    if (this.isMobile || this.props.fullscreen) {
+      this.props.controller.startHideControlBarTimer();
+    }
+  },
+
+  componentWillUpdate: function(nextProps) {
+    if(nextProps) {
+      if (nextProps.controller.state.controlBarVisible == false && this.state.controlBarVisible == true) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace "==" with "===". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AEqEqEq)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace "==" with "===". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AEqEqEq)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1125)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1125)